L1Trigger/L1THGCal/plugins/be_algorithms/SingleCellClusterAlgo.cc

```diff
@@ -12,14 +12,17 @@ class SingleCellClusterAlgo : public Algorithm<HGCalTriggerCellBestChoiceCodec>
 
         SingleCellClusterAlgo(const edm::ParameterSet& conf):
             Algorithm<HGCalTriggerCellBestChoiceCodec>(conf),
-            cluster_product_( new l1t::HGCalTriggerCellBxCollection ){}
+            cluster_product_( new l1t::HGCalTriggerCellBxCollection ),
+            calibration_(conf){}
```

Comment:
Use `conf.getParameterSet("calib_parameters")` instead of just `conf`. This way you pass only the calibration parameters.